manifests/t/Taiizor/SucroseWallpaperEngine/24.7.13.0/Taiizor.SucroseWallpaperEngine.locale.en-GB.yaml

```diff
@@ -0,0 +1,32 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: Taiizor.SucroseWallpaperEngine
+PackageVersion: 24.7.13.0
+PackageLocale: en-GB
+Publisher: Taiizor
+PublisherUrl: https://github.com/Taiizor
+PublisherSupportUrl: https://github.com/Taiizor
+PrivacyUrl: https://github.com/Taiizor/Sucrose/blob/develop/PRIVACY_POLICY.md
+Author: Taiizor
+PackageName: Sucrose Wallpaper Engine
+PackageUrl: https://github.com/Taiizor/Sucrose

```

Comment:
From the package's GitHub, they link directly to the Microsoft Store. This means the application can already be installed using `winget install xp8jgpbhtjglcq` 
```
PS C:\Users\Trenly> winget search "Sucrose"
Name                     Id             Version Source
--------------------------------------------------------
Sucrose Wallpaper Engine XP8JGPBHTJGLCQ Unknown msstore
```

![image](https://github.com/user-attachments/assets/6b7e97c0-0284-4ae1-83e0-868bc2426205)

Reply:
Should this be a blocker? I believe other packages can be installed through both sources.